packages/expo-updates/android/src/main/java/expo/modules/updates/loader/FileDownloader.kt

```diff
@@ -143,6 +144,7 @@ open class FileDownloader(private val client: OkHttpClient) {
             when (contentDispositionName) {
               "manifest" -> manifestPartBodyAndHeaders = Pair(output.toString(), headers)
               "extensions" -> extensionsBody = output.toString()
+              "certificate_chain" -> certificateChainString = output.toString()
```

Comment:
Related discussion point on the server PR - what casing do we want to use for form fields?

Reply:
I don't think it matters as long as we're consistent (spec for multipart doesn't prescribe a casing since it's the user name on the form `<input` field that this is gotten from traditionally).

I'll update the spec to say `snake_case` since that was my first gut feeling, though I'm actually not sure these need to be part of the spec since they're specific to our implementation of the spec only (for use in Expo Go).